.vuepress/config.js

```diff
@@ -14,8 +14,9 @@ module.exports = {
     editLinks: true,
     editLinkText: 'Proposer des modifications sur cette page',
     nav: [
-      { text: 'Sommaire', link: '/accueil.html' },
-      { text: 'Contacter Etalab', link: 'https://etalab.gouv.fr/contact' }
+      { text: '📚 Sommaire', link: '/accueil.html' },
+      { text: '💾 Télécharger les guides', link: '/pdf.html' },
+      { text: '✉️ Contacter Etalab', link: 'https://www.etalab.gouv.fr/contact' }
```

Comment:
Added emojis in the navbar for clarity. I can 🗑️  this if you prefer.
![image](https://user-images.githubusercontent.com/295709/89656581-ca121500-d899-11ea-871e-da956c052078.png)